lib/js-api/src/to-deprecate/alfresco-api-type.ts

```diff
@@ -34,6 +34,7 @@ export interface AlfrescoApiType {
     gsClient: LegacyHttpClient;
     authClient: LegacyHttpClient;
     processAuth: LegacyHttpClient;
+    hxiConnectorClient: LegacyHttpClient;
```

Comment:
Just curious but should we be adding stuff here, if its marked to be deprecated?

Reply:
For now to be able to use existing mechanisms I think we do until the better alternative will be introduced, I tried to implement it in some other way but I didn't find anything working